packages/gatsby-theme-aio/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/gatsby-theme-aio",
-  "version": "4.14.10",
+  "version": "4.14.11",
```

```suggestion
  "version": "4.14.12",
```

if https://github.com/adobe/aio-theme/pull/1620/files#diff-13569b8d8fcc7a79f7401b4f7b24fc104f89b0cd7352614e4f237e9370d5d2f6R3 gets merged first